_fellows/natasha-bansgopaul.md

```diff
@@ -7,14 +7,14 @@ project: []
 agency: ["NIST"]
 hometown: New York, NY
 region: east
-bio_short: Global FinTech leader and strategist with expertise in cross functional management, communication, and business development.
+bio_short: Global FinTech leader with expertise across artificial intelligence, emerging technologies, and inter-agency collaboration.
 tags: ['east', '2021', 'Strategy_And_Growth', 'Venture_Capital_Private_Equity', 'Marketing_And_Communications']
 skills: ['Strategy and Growth', 'Venture Capital/Private Equity', 'Marketing and Communications']
 ---
-As the Co-Founder and COO of DarcMatter (DM), Konstellation, and VegaX Holdings, Natasha is an experienced fintech, marketing, strategy, and M&A professional with over 10+ years of demonstrated leadership experience via Fortune100’s and startups.  As a fintech founder, she has increased the platform’s assets under management to $7.0 Bn, with users from 65+ countries.
+Natasha is an Entrepreneur in Residence (EIR) at the National Institute of Standards and Technology (NIST) where she leads A.I. Transparency initiatives as a PIF. In this role, she leads the development of the AI Transparency playbook as a part of NIST's AI Risk Management Framework (RMF), which was released in 2023. At NIST, she also supports the National Artificial Intelligence Advisory Committee's (NAIAC) which is tasked with advising the President and the National AI Initiative Office on topics related to the National AI Initiative.
 
-Natasha holds an MBA from The Pennsylvania State University (PSU), and has successfully led her companies to secure numerous awards globally such as, “Startup of the Year,” NextMoney Global FinTech Finals, HFM Technology “Best FinTech Solution for Hedge Funds,” and “Best Blockchain Technology Provider” in the 2019 from WealthManagement.com.
+Natasha Bansgopaul is a serial entrepreneur and technologist focusing on the intersection of emerging technologies and traditional infrastructure. She is the Co-Founder and COO of VegaX Holdings, an award-winning financial  technology platform building cryptocurrency index technology and automated portfolio rebalancing  infrastructure solutions for traditional institutions and investors. Natasha is an experienced financial technology, marketing, research, and M&A professional with more than 10 years of demonstrated  leadership through key strategic roles at Fortune100 companies.
```

Comment:
May be an extra space between "rebalancing" and "infrastructure" as well as "demonstrated" and "leadership"

Add a space: "Fortune 100"